site/packages/aa-signers/particle/getAuthDetails.md

```diff
@@ -0,0 +1,40 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: ParticleSigner • getAuthDetails
+  - - meta
+    - name: description
+      content: Overview of the getAuthDetails method on ParticleSigner
+  - - meta
+    - property: og:description
+      content: Overview of the getAuthDetails method on ParticleSigner
+---
+
+# getAuthDetails
+
+`getAuthDetails` returns the details about the authenticated user, specifically all EOA addresses tied to the user's Particle vault.
```

Comment:
Vault isn't usually a word we tend to use, "account" or "wallet" is usually okay. And `getAuthDetails`, which wraps `getUserInfo` returns social login info alongside uuid, user token, and their EOA addresses (EVM + Solana)